online-training.md

```diff
@@ -13,20 +13,49 @@ training as well as specific tips on using [Zoom](https://zoom.us/).
   - Decide instructor and backup-instructor in case of network issues
   - Decide helpers. One helper should be responsible for monitoring
     Zoom, i.e. the chat window, hand-raising and other feedback
-
 - TODO: set up pre-lesson polling? (zoom feature) Maybe unnecessary in view of pre-workshop survey
 - Instructors and helpers should use a reliable camera and microphone.
   Computer microphone might not be enough since audio quality will depend on
   instructor's head angle and proximity to screen.
 
+
+
+### At the beginning of the session
+
+- Allow time at the beginning of the session to debug video/audio and to
+  arrange windows. This takes few minutes so better do not start with teaching
+  from minute 1.
+- We cannot assume that all Zoom participants have the same and up to date
+  client and some clients do not contain "sticky notes" feedback or a button to
+  raise hands so agree with participants on signals (e.g. typing `\hand` in the
+  chat window seems to be standard).-
+
+
+### Recording of sessions
+
+If you plan to record the session, make sure that everybody is aware that the
+sessions is recorded, informed about how the recording will be used, and gives
+consent to be recorded:
+https://support.zoom.us/hc/en-us/articles/360026909191-Consent-to-be-Recorded
```

Comment:
Let's merge this for now but do some experiments to see how it works - or have you already?

It might record but just blank out their picture, which is a pretty good default.

Reply:
I haven't tested this yet.

Reply:
But let's merge and iterate/adjust as we learn more.